modules/ipi-install-bmc-addressing-for-cisco-cimc.adoc

```diff
@@ -0,0 +1,63 @@
+// This is included in the following assemblies:
+//
+// installing/installing_bare_metal_ipi/ipi-install-configuration-files.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="bmc-addressing-for-cisco-cimc_{context}"]
+= BMC addressing for Cisco CIMC
+
+The `address` field for each `bmc` entry is a URL for connecting to the {product-title} cluster nodes, including the type of controller in the URL scheme and its location on the network.
+
+[source,yaml]
+----
+platform:
+  baremetal:
+    hosts:
+      - name: <hostname>
+        role: <master | worker>
+        bmc:
+          address: <address> <1>
+          username: <user>
+          password: <password>
+----
+<1> The `address` configuration setting specifies the protocol.
+
+For Cisco UCS UCSX-210C-M6 hardware, Red Hat supports Cisco Integrated Management Controller (CIMC).
+
+.BMC address format for Cisco CIMC
+[width="100%", cols="1,3", options="header"]
+|====
+|Protocol|Address Format
+|Redfish virtual media| `redfish-virtualmedia://<server_kvm_ip>/redfish/v1/Systems/<serial_number>`
+|====
+
+To enable Redfish virtual media for Cisco UCS UCSX-210C-M6 hardware, use `redfish-virtualmedia://` in the `address` setting. The following example demonstrates using Redfish virtual media within the `install-config.yaml` file.
```

Comment:
Colon since it a "the following example" lead-in?
```suggestion
To enable Redfish virtual media for Cisco UCS UCSX-210C-M6 hardware, use `redfish-virtualmedia://` in the `address` setting. The following example demonstrates using Redfish virtual media within the `install-config.yaml` file.
```
Maybe throughout the PR, too, for any sentence leading into the block/example it's describing?